realm/realm-library/src/main/java/io/realm/RealmObservable.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2017 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm;
+
+/**
+ * A class implementing this interface is capable of reporting when the data stored by the class has changed. When that
```

Comment:
"has" -> "have" (I think "data" is plural in this context: https://en.wikipedia.org/wiki/Data_(word))